spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -288,6 +288,11 @@ sealed class SpdxSingleLicenseExpression : SpdxExpression() {
      * Return the license exception identifier if this is a [SpdxLicenseWithExceptionExpression] or null otherwise.
      */
     abstract fun exception(): String?
+
+    /**
+     * Return the URL to https://spdx.org for this license.
```

Comment:
Please document under which circumstances `null` is returned.

Reply:
Done.